tests/test-webapps/test-owb-cdi-webapp/src/main/java/org/eclipse/jetty/cdi/owb/OwbServletContainerInitializer.java

```diff
@@ -0,0 +1,55 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under
+// the terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0
+//
+// This Source Code may also be made available under the following
+// Secondary Licenses when the conditions for such availability set
+// forth in the Eclipse Public License, v. 2.0 are satisfied:
+// the Apache License v2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.cdi.owb;
+
+import java.util.Set;
+import javax.servlet.ServletContainerInitializer;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletException;
+
+import org.apache.webbeans.servlet.WebBeansConfigurationListener;
+
+/**
+ * @deprecated This class will not be required once https://issues.apache.org/jira/browse/OWB-1296 is available
```

Comment:
According to the status of OWB-1296, it's now resolved and available since 2.0.12. Since you upgraded `openwebbeans-jetty9` from 2.0.11 to 2.0.15, is that class still required?

Reply:
It is still required, the upgrade didn't fix anything.

This class is copied from Jetty 9.4.x at ...

https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/tests/test-webapps/test-owb-cdi-webapp/src/main/java/org/eclipse/jetty/cdi/owb/OwbServletContainerInitializer.java

.. complete with that comment and issue link.

Reply:
Then the depreciation comment is wrong and should be removed IMHO.

Reply:
I removed this hack.